### DISCUSSION TITLE
Runners / GitHub-hosted runners timers tick forever

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#12011] [Actions] [answered]
https://github.com/organizations/<org>/settings/actions/hosted-runners

Taken when I started filing opening this feedback:
![image](https://user-images.githubusercontent.com/2119212/155444949-e2731dc1-062c-407c-b14b-5805097435a6.png)

Links from each of the "Active jobs"
![image](https://user-images.githubusercontent.com/2119212/155445039-4a465ca6-6165-4dbb-88f0-d83fb7e6078a.png)
![image](https://user-images.githubusercontent.com/2119212/155445084-3e3ebba4-9cec-4e38-a416-a3aacd16bce7.png)
![image](https://user-images.githubusercontent.com/2119212/155445136-a494087d-9ddf-4828-a71b-30b968ad0048.png)
![image](https://user-images.githubusercontent.com/2119212/155445177-bcb036fe-1aeb-4d0a-b7ba-af765b17f308.png)

The same page again after I'd taken screen shots of the above:
![image](https://user-images.githubusercontent.com/2119212/155445225-eef34f17-4602-4375-be71-fee5576bfc0d.png)

Expected results: Don't tell me that jobs are running when they've been stopped for ~8 minutes or more.

![image](https://user-images.githubusercontent.com/2119212/155445320-3e75bd1d-77b8-4b41-b639-0255d011246d.png)

**Comment by ethomson** [comment]
Thanks @jsoref , we'll take a look.

**Comment by nebuk89** [comment, marked as answer]
@jsoref we have added this to the backlog to fix as we have used the same logic in a couple of places which results in the timer on the page syncing on load but not getting refreshed without a page re-load (so if you leave it open it will just keep ticking as you saw!) I will update once we have the fix going out :)
